plzcrawler/plzCrawler.py

```diff
@@ -143,16 +151,16 @@ def makePlzDataframe(self):
         plzDf = pd.DataFrame(plzDict)
         return plzDf
 
-    # neue Tabelle speichern
+    # save new table
     def savePlzTable(self, inputDfName):
-        """nimmt pandas-dataframe;
-        speichert pandas dataframe unter ursprünglichem Tabellennamen (mit "+PLZ") in Form einer .xlsx-Tabelle"""
+        """takes pandas dataframe;
+        saves pandas dataframe under the original table name (with "+zip code") in the form of an .xlsx table"""

```

Comment:
I think in this case "+PLZ" must stay as it is. Because that's the suffix which will be appended to table name.